iron-image.html

```diff
@@ -77,6 +79,12 @@
       position: relative;
     }
 
+    #img {
+      display: block;
```

Comment:
The problem is that there's space left for characters' descenders if the image is inline-block. (Though, maybe that's just a result of leftover whitespace in the template?) The host is inline-block, which should cause descendant block elements to collapse to their content's width.